### DISCUSSION TITLE
Actualizacion package.json de @tiendanube/components, @nimbus-ds/menubutton y @nimbus-ds/patterns

### DISCUSSION THREAD

**Discussion by nachozullo** [TiendaNube/nimbus-design-system#170] [Q&A] [answered]
Buenas, cómo estan?

Desde new admin estamos haciendo una migración de create-react-app a vite. Para esto encontramos que los paquetes `@tiendanube/components`, `@nimbus-ds/menubutton` y `@nimbus-ds/patterns` tienen un error en el campo `module` del archivo `package.json`.

Las modificaciones que necesitamos serían: 

![image](https://github.com/TiendaNube/nimbus-design-system/assets/28366930/d89bb153-9447-462b-91f1-58e181ba9535)
![image](https://github.com/TiendaNube/nimbus-design-system/assets/28366930/3f4eb999-a3b2-41f2-8d7f-ceccc90c45ad)
![image](https://github.com/TiendaNube/nimbus-design-system/assets/28366930/3fba2e8a-24e7-47ce-ace0-82a5d9db7403)

Las podemos hacer? Creen que hay algo que nos pueda bloquear? 
Gracias!

**Comment by juniorconquista** [comment, marked as answer]
Bom dia @nachozullo alterações relalizadas conforme a necessidade de vocês para pegar os updates basta instalar as versões: `@nimbus-ds/menubutton@1.5.2` e `@nimbus-ds/patterns@1.4.2` no caso do pacote do `menubutton` ele pode ser removido uma vez que ja esta contido no pacote `@nimbus-ds/patterns` é so mudar a importação no projeto de vocês e para o pacote `@tiendanube/components@4.0.1`.
